docs/example.xhtml

```diff
@@ -0,0 +1,71 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.1//EN' 'http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd'>
+<html xmlns='http://www.w3.org/1999/xhtml'>
```

Comment:
off-topic:  Could please create a `csdiff` issue to port `cshtml` to `html5` instead of `xhtml`.  Thank you!

Reply:
I was about to do just that :D